### PR TITLE
lumper: add support for passing a kernel command line through the CLI

### LINKED ISSUE #5: Add support for passing a kernel command line through the CLI
`./lumper --kernel vmlinuz --cmdline "console=ttyS0 i8042.nokbd reboot=k panic=1 pci=off"` and the `vmlinuz` kernel would use the passed command line.

### PR DESCRIPTION
**Pull request by sylvain-reynaud**
closes #5 

Ready for reviews :)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,12 +1,13 @@
 [package]
+edition = "2021"
 name = "lumper"
 version = "0.1.0"
-edition = "2021"
 
 [dependencies]
 clap = "3.0.0-beta.5"
-vmm = { path = "src/vmm" }
+linux-loader = {version = "0.4.0", features = ["bzimage", "elf"]}
```
Comment (sameo) [inline Cargo.toml:8]: We're going to end up with potentially 2 crates for `linux-loader`. We should only use the one that's already there.

**Review by sameo — COMMENTED** [review]
Looks pretty good to me, thanks @sylvain-reynaud. 

The `Cargo.toml` changes may need so me fixes. Also, you're adding an `initramfs.img` to the PR and I dont think you want that.

**Comment by sameo** [comment]
@sylvain-reynaud Please *rebase* this PR on top of the main branch, don't merge main into it. I want to have a linear git history unless absolutely needed.

**Comment by sylvain-reynaud** [comment]
`initramfs.img` removed

`linux-loader` removed and replaced by `vmm`

**Comment by sylvain-reynaud** [comment]
`initramfs.img` removed

`linux-loader` removed and replaced by `vmm`

File: Cargo.toml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 [package]
+edition = "2021"
 name = "lumper"
 version = "0.1.0"
-edition = "2021"
 
 [dependencies]
 clap = "3.0.0-beta.5"
-vmm = { path = "src/vmm" }
+vmm = {path = "src/vmm"}
```
Comment (sameo) [inline Cargo.toml:8]: Please remove those changes from this PR, they only add noise.

File: Cargo.toml [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
 [patch.crates-io]
 # TODO: Using this patch until a version > 4.0 gets published.
-linux-loader = { git = "https://github.com/rust-vmm/linux-loader.git", rev = "9a9f071" }
+linux-loader = {git = "https://github.com/rust-vmm/linux-loader.git", rev = "9a9f071"}
```
Comment (sameo) [inline Cargo.toml:12]: ditto.

**Review by sameo — COMMENTED** [review]
I only have 2 nitty comments @sylvain-reynaud 
Then we can merge that PR, thanks.

**Comment by sylvain-reynaud** [comment]
Done, thank you for your comments :) @sameo

**Comment by sylvain-reynaud** [comment]
Done, thank you for your comments :) @sameo

**Review by sameo — APPROVED** [review]

**Pull request merged**